### PR TITLE
Update minimatch

### PR DESCRIPTION
**Pull request by delphiactual**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,11 +42,11 @@
   "main": "./src/jshint.js",
 
   "dependencies": {
-    "cli":                 "0.6.x",
+    "cli":                 "0.11.x",
     "console-browserify":  "1.1.x",
     "exit":                "0.1.x",
     "htmlparser2":         "3.8.x",
-    "minimatch":           "2.0.x",
+    "minimatch":           "^3.0.2",
```
Comment (jugglinmike) [inline package.json]: Can you use the `~` specifier instead?

**Comment by jugglinmike** [comment]
Hi @delphiactual! Are you able to sign the CLA? No worries if not; I can follow up with a commit of my own to address this. I'd much rather have your contribution though!

**Comment by delphiactual** [comment]
@jugglinmike I did sign the CLA, so I'm not sure why that check is failing.

**Comment by jugglinmike** [comment]
I just verified your signature manually. This looks like a problem with CLAHub's
integration with GitHub, so no further action is required on your part. (In
case you're curious, I've opened an issue within that project to discuss a
solution: https://github.com/clahub/clahub/issues/135.) Thanks for the
contribution; I'll review minimatch's change log and merge this ASAP!

**Comment by jugglinmike** [comment]
Okay, I've had a chance to review the relevant change logs, and everything looks good. I've squashed this to a single commit, updated the message to satisfy [our contribution guidelines](https://github.com/jshint/jshint/blob/master/CONTRIBUTING.md), and merged to to `master` at commit 05d7a317f910e2565f079df544030c87512cdd58. Thanks for your help!

**Comment by ntwb** [comment]
@jugglinmike Is there a new minor patch release planned with this change? (i.e. From a security perspective)

**Comment by jugglinmike** [comment]
Yes--see gh-2952 for more information.

**Pull request closed without merging**